presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctionsBase.java

```diff
@@ -265,6 +265,7 @@ public void testTimeZone()
     @Test
     public void testPartFunctions()
     {
+        assertFunction("millisecond(" + TIMESTAMP_LITERAL + ")", BIGINT, (long) TIMESTAMP.getMillisOfSecond());
```

Comment:
There should be a test for each of the variants added above (interval, time w/ timezone, time, timestamp w/ timezone)